src/domain/useCase/compressUnixFiles.go

```diff
@@ -0,0 +1,25 @@
+package useCase
+
+import (
+	"errors"
+	"log"
+
+	"github.com/speedianet/os/src/domain/dto"
+	"github.com/speedianet/os/src/domain/repository"
+)
+
+func CompressUnixFiles(
+	filesQueryRepo repository.FilesQueryRepo,
+	filesCmdRepo repository.FilesCmdRepo,
+	compressUnixFiles dto.CompressUnixFiles,
+) (dto.CompressionProcessReport, error) {
+	compressionProcessReport, err := filesCmdRepo.Compress(compressUnixFiles)
+	if err != nil {
+		log.Printf("CompressUnixFilesInfraError: %s", err.Error())
+		return compressionProcessReport, errors.New("CompressUnixFilesInfraError")
+	}
+
+	log.Printf("Compressed file %s created.", compressUnixFiles.DestinationPath)
```

Comment:
file '%s'

Sempre que for mencionar um %s nas suas mensagens, é importante colocar o '' para deixar a mensagem mais fácil de ler.

Reply:
Tinha esquecido de colocar. Ajustado!

Reply:
Tinha esquecido de colocar. Ajustado!